go/database/mpt/verification.go

```diff
@@ -58,6 +55,33 @@ func VerifyFileForest(directory string, config MptConfig, roots []Root, observer
 		observer.EndVerification(res)
 	}()
 
+	// Open stock data structures for content verification.
+	observer.Progress("Obtaining read access to files ...")
+	source, err := openVerificationNodeSource(directory, config)
+	if err != nil {
+		return err
+	}
+	defer source.Close()
```

Comment:
This looks like a third copy of the code in the Archive and Live DB ... => to be unified.